shared/src/store/action-reducers/utils/log.ts

```diff
@@ -43,6 +96,469 @@ export function logPatient(
+                          currentSimulatedRegionOf(
+                              state,
+                              target as WithPosition
+                          ).id
+                      )
+                    : createTransferPointTag(state, transferTargetId)
+                : createHospitalTag(state, transferTargetId),
+        ],
+        `Ein ${
+            elementType === 'personnel'
+                ? (element as Personnel).personnelType
+                : (element as Vehicle).name
+        } wird in ${duration}ms von ${source.externalName} nach ${
+            transferTargetType === 'transferPoint'
+                ? (target as TransferPoint).externalName
+                : (target as Hospital).name
+        } transferiert.`,
+        transferSourceId
+    );
+}
+
+export function logTransferEdited(
+    state: Mutable<ExerciseState>,
+    elementId: UUID,
+    elementType: 'personnel' | 'vehicle',
+    oldTransferTargetId: UUID,
+    newTransferTargetId: UUID,
+    oldDuration: number,
+    newDuration: number
+) {
+    if (!logActive(state)) return;
+
+    const oldTarget = getElement(state, 'transferPoint', oldTransferTargetId);
+    const newTarget = getElement(state, 'transferPoint', newTransferTargetId);
+    const element = getElement(state, elementType, elementId);
+
+    logTransferPoint(
+        state,
+        newTransferTargetId === oldTransferTargetId
+            ? []
+            : [createTransferPointTag(state, oldTransferTargetId)],
+        `Der Transfer von ${
+            elementType === 'personnel'
+                ? (element as Personnel).personnelType
+                : (element as Vehicle).name
+        } nach ${
+            oldTarget.externalName
+        } in ${oldDuration}ms wurde geändert, und geht jetzt nach ${
+            newTarget.externalName
+        } in ${newDuration}ms.`,
+        newTransferTargetId
+    );
+}
+
+export function logTransferFinished(
+    state: Mutable<ExerciseState>,
+    elementId: UUID,
+    elementType: 'personnel' | 'vehicle',
+    transferTargetId: UUID
+) {
+    if (!logActive(state)) return;
+
+    const target = getElement(state, 'transferPoint', transferTargetId);
+    const element = getElement(state, elementType, elementId);
+
+    logTransferPoint(
+        state,
+        [],
+        `Der Transfer von ${
+            elementType === 'personnel'
+                ? (element as Personnel).personnelType
+                : (element as Vehicle).name
+        } nach ${target.externalName} wurde abgeschlossen.`,
+        transferTargetId
+    );
+}
+
+export function logTransferPause(
+    state: Mutable<ExerciseState>,
+    elementId: UUID,
+    elementType: 'personnel' | 'vehicle',
+    transferTargetId: UUID,
+    paused: boolean
+) {
+    if (!logActive(state)) return;
+
+    const target = getElement(state, 'transferPoint', transferTargetId);
+    const element = getElement(state, elementType, elementId);
+
+    logTransferPoint(
+        state,
+        [],
```

Comment:
You should also add a tag for the vehicle / personnel

Reply:
d